daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md

```diff
@@ -55,7 +59,8 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | emailCc | N | Output | If set this specifies the 'cc' email address of the email message. Only a single email address is allowed. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
 | emailBcc | N | Output | If set this specifies the 'bcc' email address of the email message. Only a single email address is allowed. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
 | subject | N | Output | If set this specifies the subject of the email message. Optional field, see [below](#example-request-payload) | `"subject of the email"` |
-
+| dynamicTemplateId | N | Output | If set this specifies the dynamic template id. Optional field, see [below](#example-request-payload) | `"d-123456789"` |
```

Comment:
Can dynamicTemplateId be any user defined value? Where does this Id come from, is this created by the user and can have any string format? Or is there a defined, format for the Id?

Reply:
It is the Sendgrid template id which is a string. The format is not well documented (see example of API here: https://docs.sendgrid.com/api-reference/transactional-templates-versions/retrieve-a-specific-transactional-template-version)